core/object/object.cpp

```diff
@@ -1070,14 +1070,22 @@ Error Object::_emit_signal(const Variant **p_args, int p_argcount, Callable::Cal
 
 	StringName signal = *p_args[0];
 
-	const Variant **args = nullptr;
+	// Prevent GDScript member variables being passed by pointer, see:
+	// https://github.com/godotengine/godot/issues/88885
+	LocalVector<Variant> args;
+	LocalVector<const Variant *> argptrs;
```

Comment:
Allocations here are are too bad for performance. I'd suggest using `alloca()`.

Reply:
Will do, wasn't well versed in using it when I wrote this one

Reply:
Added everywhere, might not be as relevant to the variant call one but shouldn't hurt